src/lib/modules/Document/DocumentCopyrights.js

```diff
@@ -11,8 +11,8 @@ export class DocumentCopyrights extends Component {
       <List bulleted>
         {copyrights.map((entry) => (
           <List.Item key={entry.statement}>
-            <a href={entry.url}>{entry.statement}</a> ({entry.material}) held by{' '}
-            {entry.holder}
+            <a href={entry.url}>{entry.statement}</a>{' '}
+            {entry.material && `(${entry.material})`} held by {entry.holder}
```

Comment:
in which cases is that `undefined`?

Reply:
material is not a required field of a Document's "copyright" property